README.md

```diff
@@ -1 +1,39 @@
-# opendor.me
\ No newline at end of file
+# opendor.me
+
+This is the repository for the [opendor.me](http://opendor.me). The code is entirely open source and licensed under [CCLC Licensce](LICENSE.md). Read the installation guide below to get started with setting up the app on your machine.
+
+## Requirements
+
+The following tools are required in order to start the installation.
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker-Compose](https://docs.docker.com/compose/install/)
+- [Make](https://www.gnu.org/software/make/)
+
+## Installation
+
+1. Clone this repository with `git clone https://github.com/Astrotomic/opendor.me.git`
+2. Run `cp .env.example .env` to copy .env file
+3. Run `make build-env` to build local environment
+4. Run `make start-env` to start the application
```

Comment:
I think here are steps missing like `artisan key:generate`, `composer install`, `npm install` and `npm run dev`.
I'm not sure if we could/should put them in the `make build` command @bhushan ? 🤔

Reply:
+1 I think we should add that in build itself.

Reply:
in my opinion then there should be `make setup` which can be run only once and it will do all the jobs needed to setup project .. for example in future we might add mutagen support for faster file sync, these operations could be specific to OS, `make setup` seems right place to add/improve/remove such things regarding setup. 

`make build` is intended to build docker-images only.

Reply:
also every-time i want to build image, I dont want to install composer or npm or generate key or build assets

Reply:
Sounds good! 🙂👍 Go for `make setup` 🚀